net-misc/streamlink/streamlink-0.9.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+PYTHON_REQ_USE='xml(+),threads(+)'
+
+inherit distutils-r1
+
+DESCRIPTION="CLI for extracting streams from websites to a video player of your choice"
+HOMEPAGE="https://streamlink.github.io/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="BSD-2 Apache-2.0"
+SLOT="0"
+IUSE="doc test"
+
+RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	virtual/python-futures[${PYTHON_USEDEP}]
+	virtual/python-singledispatch[${PYTHON_USEDEP}]
+	dev-python/backports-shutil_which[$(python_gen_usedep 'python2*')]
+	dev-python/backports-shutil_get_terminal_size[$(python_gen_usedep 'python2*')]
```

Comment:
you probably want `$(python_gen_cond_dep dev-python/backports-shutil_get_terminal_size[${PYTHON_USEDEP}] 'python2*')` here